### ISSUE TITLE
How to configure for multiple users with location follow

### ISSUE THREAD

**Issue by hightechstl** [AHAAAAAAA/PokemonGo-Map#3105]
I'm using the current stable release and im interested to learn how i can configure it to allow multiple people topeople to use the maps follow location feature simultaneously. As of right now if i turn off fixed location, only one person on the map will be able to see the scan for their location, so i just have fixed locations in multiple areas.

Map running in ubuntu 14.04 vps, apache2

**Comment by Paralysing** [comment]
Multiple users are currently not supported, however you could run multiple instances, 1 per user.

**Comment by hightechstl** [comment]
Ah alright. I tried to find it in the release note but thought i might just
be blind from lack of sleep. I've been considering multiple instences.
Thank you.

**Comment by dbendure** [comment]
I just host for my area, I gave a few of my local friends the link. We end up fighting over area control. I feel like you need a couple of dummy accounts to run more than three at a time. On my side anything above 4 threads kills it.

**Comment by dbendure** [comment]
If you run -os, when you drag and drop a new location it wont update the area. So I have the server running on one and a stationary one that always update a single area.

![image](https://cloud.githubusercontent.com/assets/3527843/17317434/b4188270-584b-11e6-8e80-27f7cf7513fd.png)

**Comment by hightechstl** [comment]
Yeah right now i run -os with a fixed location and then multiple other
fixed locations depending on where my friends request it at.

What do you guys mean when you are talking about "threads"? I've not
adjusted this in the config yet

**Comment by xyywy** [comment]
I did it on Win server 2008. All you need to do is to make a copy of the program folder and rename the new folder. And run the program in the new folder with a new Port number. So one person uses one port number. I am running 3 programs at the same time for my friends and me. I think you can do the same in Linux.

**Comment by xyywy** [comment]
Be careful that you should use different google or ptc account and different google map api for different programs.

**Comment by hightechstl** [comment]
So basically make a "copy" of the original folder the map is installed to and run it with different username/password/api ?

**Comment by xyywy** [comment]
Yes. it works on win server 2008.

Get Outlook for iOS




On Tue, Aug 2, 2016 at 4:13 PM +0200, "hightechstl" <notifications@github.com> wrote:





So basically make a "copy" of the original folder the map is installed to and run it with different username/password/api ?

---

**Comment by hightechstl** [comment]
What will outlook for iOS do for me????

**Comment by xyywy** [comment]
There is an outlook app for iOS don't you know?

Get Outlook for iOS




On Tue, Aug 2, 2016 at 4:18 PM +0200, "hightechstl" <notifications@github.com> wrote:





What will outlook for iOS do for me????

---

**Comment by hightechstl** [comment]
oh, I am assuming that is your signature lol. NVM... It says "Get Outlook for iOS" I thought that was part of your response to my question LOL

**Comment by xyywy** [comment]
😂😂😂no it is not my signature. I can delete it.

Get Hahaha




On Tue, Aug 2, 2016 at 4:37 PM +0200, "hightechstl" <notifications@github.com> wrote:





oh, I am assuming that is your signature lol. NVM... It says "Get Outlook for iOS" I thought that was part of your response to my question LOL

---

**Comment by tainaan** [comment]
@xyywy Personally I have only one folder but just have 3 batch files with different ports and log in info. MySQL is set up in the config.ini so they all fill that up and every map will show all of the locations being used but each port will have control of different accounts.

**Comment by xyywy** [comment]
Hi, that is a very briliant solution!! You setup your MySql in config.ini and log in three accounts in command with different ports! And you make all these accounts scanner display on one google map window! And did you use all these accounts with one google map api?
I will try as what you did. As I never use and have no idea about MySQL. I need to google the MySQL toturial first.
Thank you!



Date: Tue, 2 Aug 2016 09:41:01 -0700

**Comment by xyywy** [comment]
Hi, I have installed a MySQL on my win server 2008. But I do not know how to use that. Could you tell me how you make a database for the PokemonGoMap? Just in brief is Ok. I can also learn to do it myself. Thanks a lot!



Date: Tue, 2 Aug 2016 09:41:01 -0700
